File: MY_REPOS/web-dev-notes-resource-site/core-site/other-pages/weeks/solution/directory-browsed/DIRECTORY/JS-Files/All-JS-Weeks/fizz_buzz.js
// given a number as input, return all numbers up to and including that number
// if a number is divisible by 3, replace with 'fizz', and if it's divisible by 5,
//  replace with 'buzz'
// if it's divisible by 3 and 5, replace with 'fizzbuzz'

function fizzbuzz(n) {
  let res = [];
  let num = 1;

  while (num <= n) {
    if (num % 3 === 0 && num % 5 === 0) {
      res.push('fizzbuzz');
    } else if (num % 3 === 0) {
      res.push('fizz');
    } else if (num % 5 === 0) {
      res.push('buzz');
    } else {
      res.push(num)
    }
    num++
  }
  return res.join("");
};

console.log(fizzbuzz(20))

File: _Job-Search/InterviewPractice-master/InterviewPractice-master/Python/fizzbuzz.py
def fizzbuzz():

    i = 0
    while i < 100:

        if i % 3 == 0 and i % 5 == 0:
            print("FizzBuzz")

        elif i % 3 == 0:
            print("fizz")

        elif i % 5 == 0:
            print("buzz")

        else:
            print(i)

        i += 1


def main():
    fizzbuzz()


main()
